docs/cloud/cheatsheet.md

```diff
@@ -218,4 +230,10 @@ separately from similar instances. Example, disks named
 **sda**, **sdb**, **sdc**, and so on.
 
 **Contexts**: A grouping of charts based on the types of metrics collected and visualized.
-**disk.io**, **disk.ops**, and **disk.backlog** are all contexts.
+**disk.io**, **disk.ops**, and **disk.backlog** are all contexts. -->
+
+---
```

Comment:
These `---` are probably added by mistake.

Reply:
They are line breaks, docusaurus doesn't render them, allows for the bottom of the site page (form, edit this page etc.) to be on a separate PDF page.

Reply:
We have more things on a .md then than only work in docusaurus. The printing instructions are like this too. 

Suggestion: Put some things you need in comments in the markdown and do some additional processing in learn to produce the .mdx as it needs to be, stripping the comments like we had discussed.

Reply:
`<br>` elements used instead to achieve this, @cakrit if we don't care about having those in the last page I could just remove the line breaks. I think it is smart for the user to just discard the last page

Reply:
Keep in mind, guys: one needs to adjust these new lines every time they do changes (add/remove text). It sounds a bit chore.

Reply:
Without the line breaks it looks like this 
![image](https://user-images.githubusercontent.com/70198089/232685860-81054557-0b34-4af0-91da-4fabc9cfeebe.png)

If we don't have a problem with how it looks (I don't really like it, but with the line breaks it will be empty space anyways), I will push a commit removing them.

@ilyam8 @cakrit

Reply:
Doesn't look like a problem to me, but you and @cakrit decide.